pkgs/applications/editors/vscode/extensions/vscode-utils.nix

```diff
@@ -5,6 +5,8 @@ let
     src,
     # Same as "Unique Identifier" on the extension's web page.
     # For the moment, only serve as unique extension dir.
+    vscodeExtPublisher,
+    vscodeExtName,
```

Comment:
These item are not set for some extensions not built with `extensionsFromVscodeMarketplace`. We should either extract them from `vscodeExtUniqueId` or add them to those extensions.

Reply:
Maybe all extensions should define `mktplcRef`.